nettacker/modules/vuln/puneethreddyhc_sqli_cve_2021_41649.yaml

```
@@ -2,53 +2,55 @@ info:
   name: puneethreddyhc_sql_cve_2021_41649_vuln
   author: OWASP Nettacker Team
   severity: 8
-  description: An un-authenticated SQL Injection exists in PuneethReddyHC online-shopping-system-advanced through the /homeaction.php cat_id parameter. Using a post request does not sanitize the user input.
-  reference: 
-    - https://github.com/MobiusBinary/CVE-2021-41649
+  description: An un-authenticated SQL Injection exists in PuneethReddyHC 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)